src/components/general/HtmlDisplay.vue

```diff
@@ -5,26 +5,18 @@
 
 <script setup lang="ts">
 import { getResourcesStaticUrl } from '@/lib/fews-config'
-import { useConfigStore } from '@/stores/config'
 import { computedAsync } from '@vueuse/core'
-import { computed } from 'vue'
-import { useRoute } from 'vue-router'
 
-const route = useRoute()
+interface Props {
+  url: string | undefined
```

Comment:
`url?`?

Reply:
I think this is a small difference, as in that we want to enforce the user of HtmlDisplay to pass the variable but whether it is defined or not is a different thing.

Reply:
Ah right, that's pretty subtle indeed... Maybe `null` is more explicit in that case?